worst.html

```diff
@@ -24,6 +24,10 @@
 
 <h2>File Downloads</h2>
 
+<p>Whilst it is possible to start a download by clicking a link with a browser under Selenium&apos;s control -
+  the Selenium API does not currently expose download progress, making it less than idea for testing downloaded files.
```

Comment:
Please avoid the use of the “Selenium API” as this can be confused with Selenium RC, which isn’t WebDriver. I suggest just dropping “Selenium” so it just reads “API”.